### PR TITLE
Bumping the version numbers in docker compose files

### LINKED ISSUE #1484: Bump versions in docker-compose files
## Expected Behaviour

Update to the latest images sources from dockerhub for the openfaas containers

## Current Behaviour

Older versions

### PR DESCRIPTION
**Pull request by freshrap6**
Signed-off-by: Douglas Jones <d.m.jones2@gmail.com>

## Description

Closes #1484 

## Motivation and Context

Updating to the latest version

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
- [ ] My issue has received approval from the maintainers or lead with the `design/approved` label

## How Has This Been Tested?

Started up swarm containers on my local computers

Started Docker Swarm on my mac, brought up the UI, logged in, and deployed Colorise

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [X] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: docker-compose.arm64.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
     gateway:
         ports:
             - 8080:8080
-        image: openfaas/gateway:0.18.11-arm64
+        image: openfaas/gateway:0.18.13-arm64
```
Comment (Waterdrips) [inline docker-compose.arm64.yml]: no 0.18.13 exists for arm64.

File: docker-compose.arm64.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ services:
 
     # auth service provide basic-auth plugin for system APIs
     basic-auth-plugin:
-        image: openfaas/basic-auth-plugin:0.18.11-arm64
+        image: openfaas/basic-auth-plugin:0.18.13-arm64
```
Comment (Waterdrips) [inline docker-compose.arm64.yml]: no 0.18.13 for arm64 again :thinking:

File: docker-compose.arm64.yml (hunk 1 above)
Comment (alexellis) [reply docker-compose.arm64.yml]: That's incorrect then and shouldn't be changed

Comment (freshrap6) [reply docker-compose.arm64.yml]: Updated

File: docker-compose.arm64.yml (hunk 2 above)
Comment (freshrap6) [reply docker-compose.arm64.yml]: Updated

**Comment by alexellis** [comment]
I will merge and then fix up.

**Comment by ragingpastry** [comment]
This image doesn't exist: openfaas/queue-worker:0.8.5 which makes deployments break

**Comment by Waterdrips** [comment]
It should be 0.9.0

**Comment by freshrap6** [comment]
My apologies. I'll be more careful in the future

**Pull request merged**